public/example_templates/netjson-clustering.html

```diff
@@ -68,9 +75,13 @@
     </style>
   </head>
   <body>
+    <div class="netjsongraph-container">
+     <div id="graphChartContainer"></div>
+    </div>
```

Comment:
Can we shorten this a bit, eg?

```suggestion
    <div id="njg">
    </div>
```

The `njg` acronym stands for netjsongraph, it's already used elsewhere in the code.